qwikidata/json_dump.py

```diff
@@ -30,13 +31,13 @@ def __init__(self, filename: str) -> None:
 
         if filename.endswith(".json"):
             self.basename, _ = os.path.splitext(filename)
-        elif filename.endswith(".json.bz2"):
+        elif filename.endswith((".json.bz2", "json.gz")):
             self.basename, _ = os.path.splitext(os.path.splitext(filename)[0])
         else:
-            raise ValueError('filename must end with ".json.bz2" or ".json"')
+            raise ValueError('filename must end with ".json.bz2" or "json.gz" or ".json"')
 
         self.filename = filename
-        self.compressed = filename.endswith("json.bz2")
+        self.compressed = filename.endswith((".json.bz2", "json.gz"))
```

Comment:
I think it makes more sense to have this variable be called something like `compression` and have its value set in the same if/else clause as `basename` with it getting one of the following values ["bz2", "gz", `None`]